akka-http-core/src/main/scala/akka/http/scaladsl/Http.scala

```diff
@@ -241,16 +243,17 @@ class HttpExt(private val config: Config)(implicit val system: ActorSystem) exte
    * Constructs a [[ClientLayer]] stage using the configured default [[ClientConnectionSettings]],
    * configured using the `akka.http.client` config section.
    */
-  def clientLayer(hostHeader: Host): ClientLayer =
-    clientLayer(hostHeader, ClientConnectionSettings(system))
+  def clientLayer(hostHeader: Host, eagerClose: Boolean): ClientLayer =
```

Comment:
why should this be configurable? HTTP is rather specific in what it needs, so we can just derive this